From ab6ae0d323b3e9bcb7cb85880412f6773072de04 Mon Sep 17 00:00:00 2001
From: "Kothapeta, BikshapathiX" <bikshapathix.kothapeta@intel.com>
Date: Thu, 23 Sep 2021 13:25:30 +0530
Subject: [PATCH] update H265 hw media_codec size limit range

Updated size limit range for H265 hw codec for encoder,
required for HEVC test failures.
Fixed min - 176x144 and max - 8192x8192.

Tracked-On: OAM-99574
Signed-off-by: Kothapeta, BikshapathiX <bikshapathix.kothapeta@intel.com>
---
 groups/codecs/configurable/media_codecs_gen11.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/groups/codecs/configurable/media_codecs_gen11.xml b/groups/codecs/configurable/media_codecs_gen11.xml
index 255e663..65adb52 100644
--- a/groups/codecs/configurable/media_codecs_gen11.xml
+++ b/groups/codecs/configurable/media_codecs_gen11.xml
@@ -223,7 +223,7 @@ Only the three quirks included above are recognized at this point:
 {{/hw_ve_vp9}}
 {{#hw_ve_h265}}
         <MediaCodec name="OMX.Intel.hw_ve.h265" type="video/hevc" >
-            <Limit name="size" min="64x64" max="8192x8192" />
+            <Limit name="size" min="176x144" max="8192x8192" />
             <Limit name="alignment" value="2x2" />
             <Limit name="block-size" value="16x16" />
             <Limit name="blocks-per-second" range="1-972000" />
-- 
2.33.0

